content/docs/command-reference/config.md

```diff
@@ -96,13 +96,27 @@ multiple projects or users, respectively.
 
 ## Configuration sections
 
-The following config sections are written by this command to the project config
-file (in `.dvc/config` by default), and they support the options below:
+The following config sections are written by this command to the appropriate
+config file (`.dvc/config` by default), supporting different config options
+within:
+
+- [`core`](#core) - main section with the general config options
+- [`remote`](#remote) - sections in the config file that describe particular
+  remotes
+- [`cache`](#cache) - contains the options that affect the project's
+  <abbr>cache</abbr>.
+- [`exp`](#exp) - options to change the default repo paths assumed by
+  `dvc exp init`
+- [`plots`](#plots) - contains an option to set custom HTML templates.
```

Comment:
- [ ] Looks like this has to be updated soon per https://github.com/iterative/dvc.org/pull/3101#pullrequestreview-834936771.